worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -49,7 +49,7 @@
 
 @Slf4j
 @Service
-@SuppressWarnings("java:S2142") //java:S2142: "InterruptedException" should not be ignored
+@SuppressWarnings({"java:S2142", "java:S3655"}) //java:S2142: "InterruptedException" should not be ignored
```

Comment:
Add what S3655 is in the comment like S2142 does.

Reply:
done